ports/esp32/boards/GENERIC_S3_SPIRAM_OCT/mpconfigboard.cmake

```diff
@@ -0,0 +1,11 @@
+set(IDF_TARGET esp32s3)
+
+set(SDKCONFIG_DEFAULTS
+    boards/sdkconfig.base
+    boards/sdkconfig.usb
+    boards/sdkconfig.ble
+    boards/sdkconfig.240mhz
+    boards/sdkconfig.spiram_sx
```

Comment:
Maybe you should copy the necessary configuration code from sdkconfig.spiram_sx to sdkconfig.spiram_oct , so you can remove the sdkconfig.spiram_sx entry in this file and avoid confusion.